Translations/en.lproj/Localizable.strings

```diff
@@ -1933,9 +1933,13 @@
 // Toggle to enable a setting for all Libraries
 "enableAllLibraries" = "Enable all libraries";
 
-// Media Access - Section Title
-// Section Title for Server User Media Access Editing
-"mediaAccess" = "Media Access";
+// Enable All Devices - Toggle
+// Toggle to enable a setting for all devices
+"enableAllDevices" = "Enable all devices";
+
+// Item Access - Section Title
+// Section Title for Server User Media / Device Access Editing
+"stringWithAccess" = "%@ Access";
```

Comment:
We shouldn't have compound string like this for localization, as inflection may be different depending on the parameter.

For example: in English to Spanish:

`User Access` -> `acceso de usuario`, while `Device Access` -> `Acceso al dispositivo`. The `User` and `Device` strings don't have the inflective `de` and `al`.

Reply:
Didn't even think about that! I've changed this out.